src/controllers/management-controllers/contract-negotiation-controller.ts

```diff
@@ -87,23 +87,12 @@ export class ContractNegotiationController {
       .then((body) => expand(body, () => new ContractNegotiationState()));
   }
 
-  async cancel(
+  async terminate(
     context: EdcConnectorClientContext,
     negotiationId: string,
   ): Promise<void> {
     return this.#inner.request(context.management, {
-      path: `/v2/contractnegotiations/${negotiationId}/cancel`,
-      method: "POST",
-      apiToken: context.apiToken,
-    });
-  }
-
-  async decline(
-    context: EdcConnectorClientContext,
-    negotiationId: string,
-  ): Promise<void> {
-    return this.#inner.request(context.management, {
-      path: `/v2/contractnegotiations/${negotiationId}/decline`,
+      path: `/v2/contractnegotiations/${negotiationId}/terminate`,
```

Comment:
do the `terminate` endpoint works also without a `reason`? that field should be mandatory, I would add it to the request body

Reply:
This was added instead of "decline" and the tests were already being skipped because this endpoint's use wasn't very clear.
I had a look at Swagger hub and in my latest commit, I added a reason and updated the tests as well. Thank you for pointing this out to me 🙏